src/schema/nmdc.yaml

```diff
@@ -2873,3 +2874,16 @@ slots:
     contributors:
       - ORCID:0009-0001-1555-1601 #Anastasiya Prymolenna
       - ORCID:0000-0002-8683-0050 #Montana Smith
+
+  bulk_elect_conductivity:
+    comments:
+      - Compatible with EMSL metadata
+    description: 
+      Electrical conductivity is a measure of the bulk soil ability to carry electric 
+      current, which is mostly dictated by the chemistry of and amount of soil water. 
+      Provide the value output of the field instrument.
+    title: bulk electrical conductivity
+    examples:
+      - value: 0.017 mS/cm
```

Comment:
@brynnz22 has been writing the examples for slots in their structured formats, as opposed to flattened strings. This flattened style is fine, but please check in with @brynnz22 if you get a chance.

Reply:
@bmeluch here is a link the linkml info for the `range` of `examples` https://linkml.io/linkml-model/latest/docs/Example/. So you can include the `value` and the `description` too. Here's an example of multiple examples for `associated_dois`: https://github.com/microbiomedata/nmdc-schema/blob/7097fc7a236628cdef21978889b0f685babdd388/src/schema/nmdc.yaml#L2749. Also, I saw that the linkml defintion says `value_description`, but that doesn't work, its just `description` for some reason.

Reply:
Thanks @brynnz22 !  The `value_description` vs `description` is an annoying case of slot name mangling. I'm glad you caught that. I'm going to raise an issue about that in the LinkML repo. It's been discussed a lot so maybe it will be left like that for the time being.

Reply:
- https://github.com/linkml/linkml/issues/1733

Reply:
I think I have broken out the example value into its structured format. Thank you @brynnz22 !